tools/cldr-apps/src/org/unicode/cldr/unittest/web/TestMisc.java

```diff
@@ -82,27 +83,26 @@ public void TestIsCoverageOrganization() {
     }
 
     public void TestGitHash() {
-        final String appsVersion = CLDRConfigImpl.getGitHashForSlug("CLDR-Apps");
-        assertNotNull("getting CLDR-Apps version", appsVersion);
-        /*
-         * TODO: fail if CLDRURLS.UNKNOWN_REVISION.equals(appsVersion))
-         * and likewise for toolsVersion
-         */
-        // if (CLDRURLS.UNKNOWN_REVISION.equals(appsVersion)) {
-        //    errln("❌ appsVersion = UNKNOWN_REVISION: " + appsVersion);
-        // }
-
-        final String toolsVersion = CLDRConfigImpl.getGitHashForSlug("CLDR-Tools");
-        assertNotNull("getting CLDR-Tools version", toolsVersion);
-
-        /*
-         * TODO: add a regression test here for CLDR_DATA_HASH
-         * Reference: https://unicode-org.atlassian.net/browse/CLDR-13582
-         */
-        // final String hash = CLDRConfig.getInstance().getProperty("CLDR_DATA_HASH");
-        // assertNotNull("getting CLDR_DATA_HASH", hash);
-        // if (hash != null && !hash.matches("[0-9a-f]+")) {
-        //    errln("❌ CLDR_DATA_HASH is not hex: " + hash);
-        // }
+        final String hash = CldrUtility.getCldrBaseDirHash();
+        assertNotNull("getCldrBaseDirHash", hash);
+        if (hash != null) {
```

Comment:
minor: it's already been asserted as not-null on the prior line, so this is an unnecessary ifdef.